### ISSUE TITLE
 Column not found: 1054 Unknown column 'cacheDriver'

### ISSUE THREAD

**Issue by ppalmeida** [rinvex/subscriptions#25]
When trying to seed the Plan, the console outputs this error:

```
  Illuminate\Database\QueryException  : SQLSTATE[42S22]: Column not found: 1054 Unknown column 'cacheDriver' in 'field list' (SQL: insert into `plan_features` (`title`, `name`, `value`, `sort_order`, `resettable_period`, `resettable_interval`, `plan_id`, `cacheDriver`, `cacheLifetime`, `updated_at`, `created_at`) values ({"en":"listing_duration_days"}, basic_news, 1, 1, 1, month, 1, , -1, 2018-03-01 19:04:29, 2018-03-01 19:04:29))

  at /Users/pedropauloalmeida/WORK/PROJECTS/WAVEWEB/SPACE-ROCKET/space-rocket/vendor/laravel/framework/src/Illuminate/Database/Connection.php: 664
  660:         // If an exception occurs when attempting to run a query, we'll format the error
  661:         // message to include the bindings with SQL, which will make this exception a
  662:         // lot more helpful to the developer instead of just the database's errors.
  663:         catch (Exception $e) {
  664:             throw new QueryException(
  665:                 $query, $this->prepareBindings($bindings), $e
  666:             );
  667:         }
  668: 
  669:         return $result;

  Exception trace:

  1   PDOException::("SQLSTATE[42S22]: Column not found: 1054 Unknown column 'cacheDriver' in 'field list'")
      /Users/pedropauloalmeida/WORK/PROJECTS/WAVEWEB/SPACE-ROCKET/space-rocket/vendor/laravel/framework/src/Illuminate/Database/Connection.php : 452

```

I  think it comes from the trait ```CacheableEloquent```. I do not know how it works and why the Plan needs it. Could you please check it out?

My factory methods are:

```php
use Rinvex\Subscriptions\Models\Plan;
use Rinvex\Subscriptions\Models\PlanFeature;

$factory->define(Plan::class, function (Faker $faker) {
    return [
        'name' => $faker->word,
        'title' => $faker->words(3),
        'description' => $faker->sentence(5, true),
        'is_active' => 1,
        'price' => $faker->randomFloat($nbMaxDecimals = 2, $min = 19.0, $max = 50.0),
        'signup_fee' => $faker->randomFloat($nbMaxDecimals = 2, $min = 1.0, $max = 10.0),
        'invoice_period' => 1,
        'invoice_interval' => 'month',
        'trial_period' => 15,
        'trial_interval' => 'day',
        'sort_order' => 1,
        'currency' => 'BRL',
        'active_subscribers_limit' => 10,
        'grace_period' => 0,
        'grace_interval' => 'day',
        'prorate_day' => 1,
        'prorate_period' => 1,
        'prorate_extend_due' => 1,
    ];
});

$factory->define(PlanFeature::class, function (Faker $faker) {
    return [
        'title' => 'listing_duration_days',
        'name' => 'listing_duration_days',
        'value' => 30,
        'sort_order' => 10,
        'resettable_period' => 1,
        'resettable_interval' => 'month'
    ];
});
```

And the seeder:

```php
public function run()
    {
        $basicPlan = factory(Plan::class)->create([
            'title' => 'Basic',
            'name' => 'basic',
            'description' => "", //
            'price' => 4.90, // Price per user <= 10 users
            'active_subscribers_limit' => 10,
            'signup_fee' => 0.00,
            'invoice_period' => 1,
            'invoice_interval' => 'month',
            'trial_period' => 15,
            'trial_interval' => 'day',
            'sort_order' => 1,
            'currency' => 'BRL',
        ]);

        $basicPlan->features()->saveMany([
            factory(PlanFeature::class)->make(['name' => 'basic_news', 'value' => 1, 'sort_order' => 0]), // notícias
            factory(PlanFeature::class)->make(['name' => 'basic_announcements', 'value' => 1, 'sort_order' => 1]), // comunicados
            factory(PlanFeature::class)->make(['name' => 'basic_capsule', 'value' => 1, 'sort_order' => 2]), // biblioteca
            factory(PlanFeature::class)->make(['name' => 'basic_book', 'value' => 1, 'sort_order' => 3]),
        ]);

        $premiumPlan = factory(Plan::class)->create([
            'title' => 'Basic',
            'name' => 'basic',
            'description' => "Plano básico", //
            'price' => 9.90, // Price per user >= 10 users
            'active_subscribers_limit' => 10000,
            'signup_fee' => 0.00,
            'invoice_period' => 1,
            'invoice_interval' => 'month',
            'trial_period' => 15,
            'trial_interval' => 'day',
            'sort_order' => 1,
            'currency' => 'BRL',
        ]);

        $premiumPlan->features()->saveMany([
            factory(PlanFeature::class)->make(['name' => 'premium_news', 'value' => 1, 'sort_order' => 0]), // notícias
            factory(PlanFeature::class)->make(['name' => 'premium_announcements', 'value' => 1, 'sort_order' => 1]), // comunicados
            factory(PlanFeature::class)->make(['name' => 'premium_capsule', 'value' => 1, 'sort_order' => 2]), // biblioteca
            factory(PlanFeature::class)->make(['name' => 'premium_book', 'value' => 1, 'sort_order' => 3]),
        ]);
    }
```

**Comment by Omranic** [comment]
Fixed 127e15db8e78767f6c6a2929c470f311f594f490 Not tested, so please test and let me know if there's any other issues.

**Comment by ppalmeida** [comment]
It works!
No ```cacheDriver``` column anymore.
